libs/upd/i18n/src/lib/translations/fr-CA.json

```diff
@@ -63,7 +63,8 @@
   "position": "Classement",
   "top10-tasks-GC": "Top 10 des tâches Enquête sur la réussite des tâches du GC",
   "task": "Tâche",
-  "tasks": "Tâches",
+  "tasks": "Tâches totales",
```

Comment:
Instead of "Tâches totales", a better translation would be: "Nombre total de tâches"